update documentation (#12)
ROCm-Developer-Tools/rocr_debug_agent: docs/index.rst

.. meta::
   :description: A library that can be loaded by ROCr to print the AMDGPU wavefront states
   :keywords: ROCdebug-agent documentation, ROCR Debug Agent documentation, rocr, ROCR

.. _index:

===============================
ROCR Debug Agent documentation
===============================

> The ROCR Debug Agent (ROCdebug-agent) is a library that can be loaded by the ROCm software runtime `(ROCR) <rocr-runtime:index>` to provide the following functionalities:

Reviewer: When I build the doc locally, that "<rocr-runtime:index> text appears in the output.  Is there another way to build the doc to have cross-docs links?

Author: Fixed.

> The ROCR Debug Agent (ROCdebug-agent) is a library that can be loaded by the ROCm software runtime :doc:`(ROCR) <rocr-runtime:index>` to provide the following functionalities:

Reviewer 2: ```suggestion
The ROCR Debug Agent (ROCdebug-agent) is a library that can be loaded by the ROCm software runtime :doc:`(ROCR) <rocr-runtime:index>` to provide the following functionaly:
```

Reviewer 3: "functionaly" -> "functionality"

Author: done

> - Print the state of all AMDGPU wavefronts that cause a queue error (such as, a memory violation, executing a ``s_trap 2``, or executing an illegal instruction).

Reviewer 2: ```suggestion
- Print the state of all AMDGPU wavefronts that cause a queue error (such as a memory violation, executing a ``s_trap 2``, or executing an illegal instruction).
```

- Print the state of all AMDGPU wavefronts by sending a SIGQUIT signal to the process using ``kill -s SIGQUIT <pid>`` command or by pressing ``Ctrl-\``, while the program is executing.

> This functionality is provided for all AMDGPUs supported by the ROCm Debugger API Library :doc:`(ROCdbgapi) <rocdbgapi:index>`.

Reviewer 2: ```suggestion
This functionality is provided for all AMD GPUs supported by the ROCm Debugger API Library :doc:`(ROCdbgapi) <rocdbgapi:index>`.
```
